src/models/generation.jl

```diff
@@ -2,17 +2,12 @@ abstract type
     Generator <: PowerSystemDevice
 end
 
+include("generation/tech_common.jl")
+include("generation/econ_common.jl")
 include("generation/renewable_generation.jl")
 include("generation/thermal_generation.jl")
 include("generation/hydro_generation.jl")
 
-#=
```

Comment:
I think I like this, but it will break PowerSimulations.

Reply:
I shouldn't because I didn't generalize this yet. Just separated it, for now, to get PowerSimulations working I don't want to touch it.